src/nvim/terminal.c

```diff
@@ -1097,7 +1097,6 @@ static void refresh_terminal(Terminal *term)
     refresh_size(term, buf);
     refresh_scrollback(term, buf);
     refresh_screen(term, buf);
-    redraw_buf_later(buf, NOT_VALID);
```

Comment:
I don't know why it was needed (it's been there since `:terminal` was first implemented).